components/EmbedPage/EmbedPage.tsx

```
@@ -1,9 +1,69 @@
+import Image from "next/image";
+import isHtml from "is-html";
+
 const EmbedPage = () => {
+  const { embedCode, embed, createdContract, embedRef } =
+    useZoraCreateProvider();
+
+  if (createdContract)
+    return (
+      <div className="size-full pointer-events-none cursor-not-allowed">
+        <div
+          dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)